web/modules/custom/pineapple_general/pineapple_general.module

```diff
@@ -44,6 +44,18 @@ function pineapple_general_field_widget_form_alter(&$element, \Drupal\Core\Form\
       }
     }
   }
+
+  $items = $context['items'];
+  $field = $items->getName();
+  /* If field name is either field_a_summary or field_r_summary, set the
+  maxlength to be 200 characters and the countdown to show. */
+  if ($field == "field_a_summary" || $field == "field_r_summary") {
+    $element['#maxlength_js'] = TRUE;
+    $element['value']['#maxlength_js'] = TRUE;
+    $maxlength_js = 200;
+    $element['value']['#attributes']['maxlength'] = $maxlength_js;
+    $element['#attributes']['maxlength'] = $maxlength_js;
+  }
```

Comment:
Wrap this code in an if statement so that it runs only on the correct/desired forms and replace double quotes with single quotes on line 52